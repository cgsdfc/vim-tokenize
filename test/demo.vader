Execute(Demo);
 call tokenize#main('./test/demo.py', './test/demo.out')
