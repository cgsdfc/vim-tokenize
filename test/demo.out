1,0-1,4:            NL             ''             
1,0-1,0:            ENDMARKER      ''             
